lua/lspconfig/configs/lsp_ai.lua

```diff
@@ -0,0 +1,27 @@
+return {
+  default_config = {
+    cmd = { 'lsp-ai' },
+    filetypes = {},
+    root_dir = nil,
+    single_file_support = true,
+    init_options = {
+      memory = {
+        file_store = vim.fn.json_decode '{}',
+      },
+      models = vim.fn.json_decode '{}',
```

Comment:
```suggestion
      memory = {
        file_store = vim.empty_dict(),
      },
      models = vim.empty_dict(),
```

Reply:
Fixed and squashed everything together :+1: